contracts/utils/FilAddress.sol

```diff
@@ -302,14 +305,14 @@ library FilAddress {
      * If _recpient is some other Filecoin-native actor, this will revert.
      */
     function sendValue(address payable _recipient, uint _amount) internal {
-        require(address(this).balance >= _amount, "Address: insufficient balance");
+        if (address(this).balance < _amount) revert InsufficientFunds();
 
         (bool success, ) = _recipient.call{value: _amount}("");
-        require(success, "Address: unable to send value, recipient may have reverted");
+        if (!success) revert InvalidAddress();
```

Comment:
Wasn't sure if this should be its own error - i guess `!success` could mean several things other than the recipient not handling `.call` (built-in actors), and in those cases, this error might be misleading?